app/src/main/java/com/teamagam/gimelgimel/app/sensor/viewModel/adapter/SensorRecyclerArrayAdapter.java

```diff
@@ -0,0 +1,112 @@
+package com.teamagam.gimelgimel.app.sensor.viewModel.adapter;
+
+import android.content.Context;
+import android.graphics.drawable.Drawable;
+import android.view.View;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.teamagam.gimelgimel.R;
+import com.teamagam.gimelgimel.app.common.base.adapters.BaseRecyclerArrayAdapter;
+import com.teamagam.gimelgimel.app.common.base.adapters.BaseRecyclerViewHolder;
+import com.teamagam.gimelgimel.app.common.base.adapters.DataRandomAccessor;
+import com.teamagam.gimelgimel.app.sensor.model.SensorMetadataApp;
+
+import butterknife.BindView;
+
+public class SensorRecyclerArrayAdapter extends
+        BaseRecyclerArrayAdapter<SensorRecyclerArrayAdapter.SensorRecyclerViewHolder, SensorMetadataApp> {
+
+
+    private Drawable mSensorDrawable;
+    private SensorMetadataApp mLastSelected;
+    private DataRandomAccessor<SensorMetadataApp> mAccessor;
+
+    public SensorRecyclerArrayAdapter(
+            DataRandomAccessor<SensorMetadataApp> data,
+            OnItemClickListener<SensorMetadataApp> listener) {
+        super(data, listener);
+        mAccessor = data;
+    }
+
+    public synchronized void select(String sensorId) {
+        unselectLast();
```

Comment:
I'm not sure you need this unselected last. unless you think it will be needed for some future use